cli/command/load_balancer_command.go

```diff
@@ -221,6 +281,78 @@ func (l *LoadBalancerCommand) Get(c *cli.Context) error {
 	return l.Printer.PrintLoadBalancers(loadBalancers...)
 }
 
+func (l *LoadBalancerCommand) HealthCheck(c *cli.Context) error {
+	args, err := extractArgs(c.Args(), "NAME")
+	if err != nil {
+		return err
+	}
+
+	id, err := l.resolveSingleID("load_balancer", args["NAME"])
+	if err != nil {
+		return err
+	}
+
+	loadBalancer, err := l.Client.GetLoadBalancer(id)
+	if err != nil {
+		return err
+	}
+
+	updateIsRequired := false
+	if target := c.String("set-target"); target != "" {
+		if split := strings.Split(target, ":"); len(split) != 2 {
```

Comment:
It does look like AWS gives some pretty decent errors - I might just leave that error handling up to them.

```
$ l0 loadbalancer healthcheck --set-target abbabba:9999999 test-lb-1
AWS Error: healthcheck target must begin with one of http, tcp, https, ssl (code 'ValidationError')

$ l0 loadbalancer healthcheck --set-target http:9999999 test-lb-1
AWS Error: healthcheck http target must specify a port followed by a path that begins with a slash. e.g. http:80/ping/this/path (code 'ValidationError')
```